src/OrchardCore/OrchardCore.Users.Core/Models/User.cs

```diff
@@ -17,6 +17,7 @@ public class User : Entity, IUser
         public string ResetToken { get; set; }
         public IList<string> RoleNames { get; set; } = new List<string>();
         public IList<UserLoginInfo> LoginInfos { get; set; } = new List<UserLoginInfo>();
+        public string TimeZone;
 
```

Comment:
Not needed if we don't implement it completely. This will just add the TimeZone=null in the database for each users.